configure.ac

```diff
@@ -457,7 +459,7 @@ AC_ARG_ENABLE([pmi-mpi], [AC_HELP_STRING([--enable-pmi-mpi],
 
 AS_IF([test "$enable_pmi_simple" = "yes" -o "$enable_pmi_mpi" = "yes"],
      [TEST_RUNNER='$(abs_top_builddir)/src/oshrun -n $(NPROCS)'],
-     [OPAL_CHECK_PMI(pmi_type)])
+     [OPAL_CHECK_PMIX(pmi_type)])
```

Comment:
Can this check return `pmi_type` as pmi1 or pmi2? If not, the checks on line 477 and 478 may not make sense any more. But, for now, seems to be ok.